finder-app/Makefile

```diff
@@ -0,0 +1,9 @@
+writer: writer.o
+	$(CROSS_COMPILE)gcc -o writer writer.o
+
+
+writer.o: writer.c
+	$(CROSS_COMPILE)gcc -c -Wall -Werror writer.c
```

Comment:
@DivyeshShashikant Why do you need to compile twice?

Reply:
I think I forgot to update it. But this won't matter right?